### ISSUE TITLE
Protected Function Error

### ISSUE THREAD

**Issue by MetalHeadSeven** [tullamods/Dominos#890]
11.0.4

Retail

1x [ADDON_ACTION_BLOCKED] AddOn 'Dominos' tried to call the protected function 'SecureStateDriverManager:SetAttribute()'.
[string "@!BugGrabber/BugGrabber.lua"]:485: in function <!BugGrabber/BugGrabber.lua:485>
[string "=[C]"]: in function `SetAttribute'
[string "@Blizzard_FrameXML/SecureStateDriver.lua"]:18: in function <...rfaceBlizzard_FrameXML/SecureStateDriver.lua:16>
[string "=(tail call)"]: ?
[string "@Dominos/core/frame.lua"]:616: in function `?'
[string "@Dominos_Progress/main.lua"]:165: in function `ForAllBars'
[string "@Dominos_Progress/main.lua"]:108: in function `?'
[string "@AdvancedInterfaceOptions/libs/CallbackHandler-1.0-8/CallbackHandler-1.0.lua"]:109: in function <...ons/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:109>
[string "=[C]"]: ?
[string "@AdvancedInterfaceOptions/libs/CallbackHandler-1.0-8/CallbackHandler-1.0.lua"]:19: in function <...ons/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:15>
[string "@AdvancedInterfaceOptions/libs/CallbackHandler-1.0-8/CallbackHandler-1.0.lua"]:54: in function `Fire'
[string "@Dominos/libs/AceEvent-3.0-4/AceEvent-3.0.lua"]:120: in function <...ceDominos/libs/AceEvent-3.0/AceEvent-3.0.lua:119>

Locals:
_ = Frame {
}
event = "ADDON_ACTION_BLOCKED"
events = <table> {
}

**Comment by Tuller** [comment]
Do you get this with just Dominos + Dominos_Config enabled?

**Comment by MetalHeadSeven** [comment]
Yes both were enabled.  Whatever the problem is reverting to 11.0.2 resolved it.  With 11.0.4 getting the error is very random it doesn't always happen and I haven't noticed a specific situation that triggers it.

**Comment by Tuller** [comment]
Yeah, taint issues like that can cause issues with Dominos (as anything that can break the stock action bars can also break Dominos' version)
